setup.py

```diff
@@ -8,7 +8,7 @@
 setup(
     name='rosinstall_generator',
     version=__version__,
-    install_requires=['argparse', 'catkin_pkg >= 0.1.28', 'rosdistro >= 0.5.0', 'rospkg', 'PyYAML', 'setuptools'],
+    install_requires=['argparse', 'catkin_pkg >= 0.1.28', 'rosdistro >= 0.6.7', 'rospkg', 'PyYAML', 'setuptools'],
```

Comment:
The version also needs to be updated in these two lines in the `stdeb.cfg` file: https://github.com/ros-infrastructure/rosinstall_generator/blob/ce4e6ec834d1c7238ebfaf7243d355bde3b5676e/stdeb.cfg#L2-L3

Reply:
Fixed.

Reply:
Conflict introduced by #45 is also resolved now.